engine/table/src/main/java/io/deephaven/engine/table/impl/join/stamp/ObjectNoExactReverseStampKernel.java

```diff
@@ -84,17 +86,7 @@ static private void computeRedirections(ObjectChunk<Object, Values> leftStamps,
     // region comparison functions
     // descending comparison
     private static int doComparison(Object lhs, Object rhs) {
-        if (lhs == rhs) {
-            return 0;
-        }
-        if (lhs == null) {
-            return 1;
-        }
-        if (rhs == null) {
-            return -1;
-        }
-        //noinspection unchecked,rawtypes
-        return ((Comparable)rhs).compareTo(lhs);
+        return ObjectComparisons.compare(rhs, lhs);
```

Comment:
What does no exact mean?

Reply:
This seem to only be applicable to `aj` / `raj` and set to true for `>` and `<` (and false for `>=` and `<=`). The refactor here should not change the behavior of this method.